docs/user-guide/configuration.md

```diff
@@ -78,9 +78,16 @@ edit_uri: src/default/docs/
 For other repository hosts, `edit_uri` works the same way. Simply specify the
 relative path to the docs directory.
 
-**default**: `blob/master/docs/` or `src/default/docs/` for GitHub or Bitbucket
+**default**: `edit/master/docs/` or `src/default/docs/` for GitHub or Bitbucket
 repos, respectively, if `repo_url` matches those domains, otherwise `null`
 
+!!! note "Note:"
```

Comment:
I think it maintains stylistic consistency with the other admonitions on the same page...but if you want me to remove it, I will.